### DISCUSSION TITLE
Bun v1.0.5

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#6411] [Announcements]
To install Bun v1.0.5:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.5:
```bash
bun upgrade
```

**[Read Bun v1.0.5's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.5)**

## What's Changed
* Add license information to package.json by @mathiasrw in https://github.com/oven-sh/bun/pull/6255
* Docs: Update context.md by @wbjohn in https://github.com/oven-sh/bun/pull/6256
* docs: update `typescript.md` by @jsparkdev in https://github.com/oven-sh/bun/pull/6222
* feat(encoding): support BOM detection with test passed by @WingLim in https://github.com/oven-sh/bun/pull/6074
* Fix bunx command for github package #5974 by @axlEscalada in https://github.com/oven-sh/bun/pull/6042
* fix(blob): Add the current offset to the new blob slice by @Hanaasagi in https://github.com/oven-sh/bun/pull/6259
* compat: Buffer: allow optional positional arguments to be undefined by @kitsuned in https://github.com/oven-sh/bun/pull/4911
* Support `bun run --if-present` by @Electroid in https://github.com/oven-sh/bun/pull/6248
* Upgrade webkit by @dylan-conway in https://github.com/oven-sh/bun/pull/6277
* Closes #6309 by @colinhacks in https://github.com/oven-sh/bun/pull/6310
* Create `AbortSignal` without JS by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6278
* Update `preload` documentation to make applicability to plugins more obvious by @Pandapip1 in https://github.com/oven-sh/bun/pull/6298
* Fix `query` property of `url` object by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6274
* fix semver and repository host bugs by @dylan-conway in https://github.com/oven-sh/bun/pull/6312
* Use singular nouns when appropriate on install messages by @jakeboone02 in https://github.com/oven-sh/bun/pull/2543
* Update Node.js compat docs by @colinhacks in https://github.com/oven-sh/bun/pull/6282
* test(bun install): cover http(s) non-github case by @booniepepper in https://github.com/oven-sh/bun/pull/6190
* WebCryptoAPI: remove JWK "EdDSA" alg checks by @panva in https://github.com/oven-sh/bun/pull/6294
* fix(serve): fix `close_connection` behaviour in `endWithoutBody` by @Hanaasagi in https://github.com/oven-sh/bun/pull/6315
* docs: `file.stream()` is not a promise by @pnodet in https://github.com/oven-sh/bun/pull/6337
* [types] allow onLoad plugin callbacks to return undefined by @silversquirl in https://github.com/oven-sh/bun/pull/6346
* Fix memory leak in fetch() by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6350
* feat(KeyObject) by @cirospaciari in https://github.com/oven-sh/bun/pull/5940
* Fix array variables preview in debugger by @2hu12 in https://github.com/oven-sh/bun/pull/6379
* fix: support uint8 exit code range by @liz3 in https://github.com/oven-sh/bun/pull/6303
* docs: fixing a couple typos by @Cadienvan in https://github.com/oven-sh/bun/pull/6331
* Update README.md by @Longju000 in https://github.com/oven-sh/bun/pull/6291
* added commands by @babarkhuroo in https://github.com/oven-sh/bun/pull/6314
* [Docs] Add Fedora build instructions to development.md by @otterDeveloper in https://github.com/oven-sh/bun/pull/6359
* fix(AbortSignal/fetch) fix AbortSignal.timeout, fetch lock behavior and fetch errors by @cirospaciari in https://github.com/oven-sh/bun/pull/6390
* Documentation for the IPC of Bun.spawn by @RaresAil in https://github.com/oven-sh/bun/pull/6400
* Update vite.md : removal of incorrect information about --bun flag by @yukulele in https://github.com/oven-sh/bun/pull/6399
* Add missing ws declarations by @vthemelis in https://github.com/oven-sh/bun/pull/6307
* docs: rearranged cli/runtime related sections by @cyfung1031 in https://github.com/oven-sh/bun/pull/6275
* Add File to binary data TOC by @AaronDewes in https://github.com/oven-sh/bun/pull/6025
* make peer dependencies install by default by @dylan-conway in https://github.com/oven-sh/bun/pull/6396
* fix: form data content type by @yadav-saurabh in https://github.com/oven-sh/bun/pull/6380
* feat(test): implement `toEqualIgnoringWhitespace` by @EladBezalel in https://github.com/oven-sh/bun/pull/6293
* Fix lifecycle scripts not running on reinstallation by @Arden144 in https://github.com/oven-sh/bun/pull/6376
* feat(install): automatically migrate package-lock.json to bun.lockb by @paperdave in https://github.com/oven-sh/bun/pull/6352

## New Contributors
* @mathiasrw made their first contribution in https://github.com/oven-sh/bun/pull/6255
* @wbjohn made their first contribution in https://github.com/oven-sh/bun/pull/6256
* @jsparkdev made their first contribution in https://github.com/oven-sh/bun/pull/6222
* @kitsuned made their first contribution in https://github.com/oven-sh/bun/pull/4911
* @Pandapip1 made their first contribution in https://github.com/oven-sh/bun/pull/6298
* @panva made their first contribution in https://github.com/oven-sh/bun/pull/6294
* @2hu12 made their first contribution in https://github.com/oven-sh/bun/pull/6379
* @Cadienvan made their first contribution in https://github.com/oven-sh/bun/pull/6331
* @Longju000 made their first contribution in https://github.com/oven-sh/bun/pull/6291
* @babarkhuroo made their first contribution in https://github.com/oven-sh/bun/pull/6314
* @otterDeveloper made their first contribution in https://github.com/oven-sh/bun/pull/6359
* @RaresAil made their first contribution in https://github.com/oven-sh/bun/pull/6400
* @yukulele made their first contribution in https://github.com/oven-sh/bun/pull/6399
* @vthemelis made their first contribution in https://github.com/oven-sh/bun/pull/6307
* @EladBezalel made their first contribution in https://github.com/oven-sh/bun/pull/6293
* @Arden144 made their first contribution in https://github.com/oven-sh/bun/pull/6376

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.4...bun-v1.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.5'>Bun v1.0.5</a>.</em>

**Comment by devgauravjatt** [comment]
**for windows exe**

**Comment by sixnaskunz** [comment]
why I can't use this command below in package.json when I upgrade to 1.0.5-debian
`
{
   "scripts": {
       "start": "NITRO_SSL_CERT=$(cat ./certificates/xxx.crt) NITRO_SSL_KEY=$(cat ./certificates/xxx.key) node .output/server/index.mjs"
   }
}`

I got error on docker container logs.
`NITRO_SSL_CERT=$(cat ./certificates/xxx.crt) NITRO_SSL_KEY=$(cat ./certificates/xxx.key) node .output/server/index.mjs
error: script "start" exited with code 6 (SIGABRT)`

How can I fix this problem?

**Reply by mathiasrw** [reply]
Please make a new issue with the information needed to replicate this.

**Reply by sixnaskunz** [reply]
Ok next time I will do but this issue already solved in version 1.0.6. Thank you.
